packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -74,7 +74,6 @@ import {
     InboundAttachMessage,
     IFluidDataStoreContextDetached,
     IFluidDataStoreRegistry,
```

Comment:
we could actually get rid of all changes in this class for similar reasons to: https://github.com/microsoft/FluidFramework/pull/11887/files#r971106783